### DISCUSSION TITLE
[BUG] : Unable to connect to mongodb connection string

### DISCUSSION THREAD

**Discussion by darshil89** [prisma/prisma#23130] [Q&A]
### Bug description

After making the prisma schema , on entering npx prisma db push , i am getting this error
![Screenshot 2024-02-14 035304](https://github.com/prisma/prisma/assets/109381405/6da2dd17-a2db-4427-be71-643ecdf378bf)

### Expected behavior

The expected behavior should be the schema should be up to date  with the changes.

### Prisma information

![Screenshot 2024-02-14 065144](https://github.com/prisma/prisma/assets/109381405/3d618bec-06ba-466a-88a4-1260af0b0913)
![Screenshot 2024-02-14 065235](https://github.com/prisma/prisma/assets/109381405/4be3a85a-1eab-449b-b6f9-f02a4c1199c5)

### Environment & setup

![Screenshot 2024-02-14 065323](https://github.com/prisma/prisma/assets/109381405/57a15aae-2c5b-4732-b122-dc14cfb2d0aa)

### Prisma Version

```

```

**Comment by jkomyno** [comment]
Hi, this looks like a problem on the hosted MongoDB side, not Prisma.
Can you please try running [MongoDB locally via Docker](https://www.mongodb.com/docs/manual/tutorial/install-mongodb-community-with-docker/), and report back? Thanks.

---

Also, we'd appreciate if in future issues you pasted the actual code snippets, rather than screenshots, in order to readily copy them to reproduce issues locally. Thank you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
